zkstack_cli/crates/zkstack/src/commands/chain/gateway/convert_to_gateway.rs

```diff
@@ -41,11 +47,40 @@ use crate::{
 lazy_static! {
     static ref DEPLOY_GATEWAY_TX_FILTERER_ABI: BaseContract =
         BaseContract::from(parse_abi(&["function runWithInputFromFile() public"]).unwrap(),);
-    static ref GATEWAY_VOTE_PREPARATION_ABI: BaseContract =
-        BaseContract::from(parse_abi(&["function run() public"]).unwrap(),);
+    static ref GATEWAY_VOTE_PREPARATION_ABI: BaseContract = BaseContract::from(
+        parse_abi(&["function prepareForGWVoting(uint256 ctmChainId) public"]).unwrap(),
+    );
 }
 
-pub async fn run(args: ForgeScriptArgs, shell: &Shell) -> anyhow::Result<()> {
+#[derive(Debug, Serialize, Deserialize, Parser)]
+pub struct ConvertToGatewayArgs {
+    /// All ethereum environment related arguments
+    #[clap(flatten)]
+    #[serde(flatten)]
+    pub forge_args: ForgeScriptArgs,
+
+    /// Pass the bridgehub, if existing ecosystem is being used
+    #[clap(long)]
+    pub bridgehub_addr: Option<Address>,
```

Comment:
we have a bridgehub inside ecosystem. you have to use it from there, if non is provided